src/components/account-tab.js

```diff
@@ -77,7 +77,7 @@ function AccountTab({ formItemLayout, tailFormItemLayout }) {
       <UsernameFormItem name="username" usernamesInUse={state.forbiddenUsernames} initialValue={user.username} {...formItemLayout} />
       <EmailFormItem name="email" emailsInUse={state.forbiddenEmails} initialValue={user.email} {...formItemLayout} />
       <FormItem {...tailFormItemLayout}>
-        <Button type="link" size="small" onClick={handleResetPasswordClick}>{t('resetPassword')}</Button>
+        <a onClick={handleResetPasswordClick}>{t('resetPassword')}</a>
```

Comment:
Just out of curiosity, why did you change the Button into an a?

Reply:
I was clicking through the pages to make sure nothing is broken or looks very bad after this styling change and I noticed that this guy was the only one that was not restyled as a link (had no underline). So I streamlined it.